### PR TITLE
WIP: Updates related to sbot discovery

### PR DESCRIPTION
**Pull request by benhylau**

### CODE REVIEW

File: ssb-broadcast.sh [hunk 1]
```diff
@@ -4,11 +4,11 @@ while true; do
     id=$(sbot whoami | grep id | awk -F "\"" '{print $4}' | sed 's/.ed25519//' | sed 's/@//')
     if ! [ -z "$id" ]; then
         for int in $(ls -1Atu /sys/class/net); do
-            ip=$(ip addr show $int | grep -v inet6 | grep -v '127.0.0.1' |grep inet | head -n 1 | awk '{print $2}' | awk -F "/" '{print $1}')
+            ip=$(ip addr show $int | grep -v inet6 | grep -v '127.0.0.1' | grep inet | head -n 1 | awk '{print $2}' | awk -F "/" '{print $1}')
```
Comment (jedahan) [inline ssb-broadcast.sh:7]: does `ip=$(ip addr show $int | grep -oE '([0-9.]+)/8'` do the same thing here?

Comment (jedahan) [reply ssb-broadcast.sh:7]: or `ip=$(ip addr show $int | grep -oE '([0-9]{1,3}\.){3}[0-9]{1,3}')` if you want a bit more sane restrictions

Comment (darkdrgn2k) [reply ssb-broadcast.sh:7]: Then you get broadcast addresses to

```
pi@raspberrypi:~ $ ip addr show $int | grep -oE '([0-9]{1,3}\.){3}[0-9]{1,3}'
192.168.1.211
192.168.1.255
```

Comment (benhylau) [reply ssb-broadcast.sh:7]: I will keep it as is for now then

**Pull request merged**